crypto/fipsmodule/ec/p384.c

```diff
@@ -886,22 +890,22 @@ static void ec_GFp_nistp384_point_mul_base(const EC_GROUP *group,
     // doubling can't happen).
     // For group (3) we process digits s_51 to s_3, for group (2) s_54 to s_2,
     // group (1) s_53 to s_1, and for group (0) s_52 to s_0.
-    const size_t start_idx = 55 - i - (i == 0 ? 4 : 0);
-    for (size_t j = start_idx; j < 55; j -= 4) {
+    const size_t start_idx = P384_MUL_NWINDOWS - i - (i == 0 ? 4 : 0);
+    for (size_t j = start_idx; j < P384_MUL_NWINDOWS; j -= 4) {
       // For each digit |d| in the current group read the corresponding point
       // from the table and add it to |res|. If |d| is negative, negate
       // the point before adding it to |res|.
       int16_t d = rnaf[j];
       // is_neg = (d < 0) ? 1 : 0
-      int16_t is_neg = (d >> 7) & 1;
+      int16_t is_neg = (d >> 15) & 1;
```

Comment:
Good catch! Thank you. There is another occurrence at line 772.
Just curious, was it causing errors in P-521?

Reply:
No, it could cause errors only with window size >7 I think. Fixed 772.